app/javascript/components/dashboard-widgets/widget-wrapper/index.jsx

```diff
@@ -80,6 +83,32 @@ const WidgetWrapper = ({
 
   return (
     <div className="card-pf card-pf-view">
+      <Modal
```

Comment:
Should we consider moving this modal to another component within the `widget-wrapper` folder to avoid congestion?

Reply:
Yeah I made the fix and repushed